chia/util/cached_bls.py

```diff
@@ -42,12 +42,12 @@ def get_pairings(
                 pk_parsed = G1Element.from_bytes(pks[i])
                 pk_bytes_to_g1[pks[i]] = pk_parsed
 
-            pairing = pk_parsed.pair(aug_hash)
+            pairing = aug_hash.pair(pk_parsed)
 
             h = std_hash(aug_msg)
             cache.put(h, pairing)
             pairings[i] = pairing
-    return pairings
+    return cast(List[G2Element], pairings)
```

Comment:
`pairings` is a list of *optional* `G2Element`s, but this function ensures they're all set, so we cast it to the correct return value

Reply:
Maybe it would be better to have a separate list hinted properly such that mypy can help us enforce this?

Reply:
reraised after modification